jee-commons/src/main/java/biz/paluch/jee/commons/BeanLookup.java

```diff
@@ -136,5 +153,14 @@ public static BeanManager beanManager() {
     public static void reset() {
         lookupStrategy = new BeanManagerLookupStrategy();
     }
+    
+    private static BeanManager beanManagerLookup() {
+        for(String jndiName : BEANMANAGER_JNDI_NAMES) {
+            BeanManager beanManager = NamingLookup.doLookup(jndiName);
+            if(beanManager != null)
+                return beanManager;
+        }
+        throw new RuntimeException("Did not found any BeanManager!");
```

Comment:
Please use a more specific exception, like IllegalStateException, not a raw type.